File: app/src/main/java/com/theah64/retrokitexample/model/data/GetMoviesData.java
package com.theah64.retrokitexample.model.data;

import com.google.gson.annotations.SerializedName;
import com.theah64.retrokitexample.model.Movie;

import java.util.List;

/**
 * Created by theapache64 on 25/8/17.
 */

public class GetMoviesData {

    @SerializedName("movies")
    private final List<Movie> movies;

    public GetMoviesData(List<Movie> movies) {
        this.movies = movies;
    }

    public List<Movie> getMovies() {
        return movies;
    }
}

File: app/src/main/java/com/theah64/retrokitexample/model/data/GetMoviesData.kt
package com.theah64.retrokitexample.model.data

import com.google.gson.annotations.SerializedName
import com.theah64.retrokitexample.model.Movie

/**
 * Created by theapache64 on 25/8/17.
 */

class GetMoviesData(@field:SerializedName("movies")
                    val movies: List<Movie>)
